samples/dashboards/demo/election/demo.js

```diff
@@ -573,34 +642,38 @@ function resetMap(mapChart) {
     }
 }
 
-function getHistoricalElectionSeries(state = 'US') {
+function getHistoricalElectionSeries(state = 'US', year) {
```

Comment:
Here we get a lint error: "Default parameter should be last". If you set year to say default '' or null this function would also be easier to understand (on line 321 it is invoked without arguments).